lambda/src/main/resources/db/migration/V5__add_checksum_to_file_property.sql

```diff
@@ -0,0 +1 @@
+insert into "FileProperty" ("PropertyId" , "Name", "Description", "Shortname") VALUES (uuid_generate_v4(), 'ServerSideChecksum', 'The checksum calculated after upload', 'Checksum');
```

Comment:
I'd make it clear in the name, description and short name that this is the SHA-256 checksum so it's unambiguous, and in case we add another checksum algorithm in future.

Reply:
Yep, makes sense. I've changed it.